backends/gstreamer/lib.rs

```diff
@@ -159,6 +216,23 @@ impl Backend for GStreamerBackend {
     fn set_capture_mocking(&self, mock: bool) {
         self.capture_mocking.store(mock, Ordering::Release)
     }
+
+    fn mute(&self, id: &ClientContextId, val: bool) {
+        let muteables = self.muteables.lock().unwrap();
+        match muteables.get(id) {
+            Some(vec) => vec.into_iter().for_each(|(_muteable_id, weak)| {
+                if let Some(mutex) = weak.upgrade() {
```

Comment:
If `upgrade` returns `None` it means that the `Mutable` has since been dropped. In that case we should remove it from the vec.